src/overcooked_demo/server/static/assets/agents/README.md

```diff
@@ -0,0 +1,68 @@
+# Example Directory for Pre-trained Rllib Agent
+
+## Structure
+
+* `agent`  
+  * `.is_checkpoint`
+  * `.tune_metadata`
+  * `checkpoint-#`
+
+Note that the file names must match EXACTLY the names and relative structure as above. The agent name is only determined and reflected in the parent directory name.
+
+In the future, the naming convenction will probably be more flexible, but for now it is very strict to simplify serialization logic
+
+## Training the Agent
+
+In general, a directory with the structure above can be trained with the human_aware_rl module in the [overcooked_ai](https://github.com/HumanCompatibleAI/overcooked_ai) repo. Specific installation instruction is available. 
```

Comment:
Not sure if it's possible to create a test that a newly trained model with human_aware_rl module will work fine in overcooked-demo. If so, we should probably add it

Reply:
I'll think about that. The idea is that as long as a trainer successfully finishes, the agent/policy it creates should by default work because the trainer and demo rely on the same underlying environment. I'll think about what are some cases that the agents might not work.

Reply:
Yeah I guess I'm mostly thinking whether it's possible to test whether there would be JS errors once the model is loaded in the browser, but that seems quite hard to do and maybe not worth it?